JoinColony/colonyCDapp
# Add support for CreateDomain action

## Description

This PR adds support for the CreateDomain action.

<img width="995" alt="image" src="https://user-images.githubusercontent.com/112586815/220772541-2fe9eb52-c0e7-4cb5-9abd-0877dbff98e2.png">

## Testing
1. Create a colony using the wizard.
2. Click on the Test Create Domain action.
3. You'll be redirected to a transaction not found page, wait a few seconds and refresh.
4. You should see the action details (including domain name, color, description). Please verify the domain is added to the domain list dropdown as well.

Resolves #279
